lib/commands/lar_city_cli/key_chain_cmd.thor

```diff
@@ -1,18 +1,18 @@
 # frozen_string_literal: true
 
 require 'fileutils'
-require_relative 'base'
+require_relative 'base_cmd'
 
 module LarCityCLI
   # Manage credentials for the rails app
-  class KeyChain < Base
+  class KeyChainCmd < BaseCmd
     class_option :environment,
                  type: :string,
-                 aliases: '-e',
+                 aliases: %w[-e --env],
                  desc: 'Environment',
                  required: false
 
-    namespace :'lx-cli:secrets'
+    namespace :'lx-cli:keys'
```

Comment:
This is an unnecessary refactor. The namespace should either be aliased or lets skip this edit.

```suggestion
    namespace :'lx-cli:secrets'
```